src/styles/NavigationBar.css

```diff
@@ -0,0 +1,77 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  padding: 4%;
```

Comment:
- [ ] Kindly remove the padding, as I can see that you did work according to the `Zeplin` design, but I think you misunderstood the design, the space around the main content is the workspace of the Zeplin, in you don't need to add any spacing around the application. So that's why if you remove this selected padding then, your application looks good.